README.md

```diff
@@ -13,6 +15,10 @@ on `poll(2)`. Implementations using `kqueue(2)`, `epoll(2)`,
 possible use various libraries for asynchronous operations like
 `libevent`, `libuv`, etc.
 
+**Implements**: [Sender/Receiver Interface for Networking(P2762)](http://wg21.link/p2762)
```

Comment:
```suggestion
**Implements**: [Sender/Receiver Interface for Networking (P2762R2)](http://wg21.link/P2762R2)
```

Reply:
oh wait -- why do you want to lock to a specific version instead of getting the latest?

Reply:
<img width="1097" alt="image" src="https://github.com/user-attachments/assets/6b5b1ab8-bbbc-4ac7-b8e6-32abeb04ab48" />

This is what the Beman Standard suggests. I think it's better this way, you may have implementation behind the design from latest paper, right? I think it's OK to specify a concrete version.

Reply:
I think this is a good suggestion since the paper may get a different revision and users may be misleads thinking that they are using the implementation of the latest revision of the paper